app/views/user_mailer/swap_confirmed.html.haml

```diff
@@ -1,31 +1,34 @@
 %p
-	Hi #{@user.name},
+  Hi #{@user.name},
 %p
-	You're all set to swap your vote with
-	= user_profile_link(@swap_with) + "!"
-	#{@swap_with.name} is going to vote for the #{@swap_with.willing_party.name} party for you, in the #{@swap_with.constituency.name} constituency.
-	In return, you will vote for the #{@swap_with.preferred_party.name} party in your constituency (#{@user.constituency.name}).
+  You're all set to swap your vote with
+  = user_profile_link(@swap_with) + "!"
+  #{@swap_with.name} is going to vote for the #{@swap_with.willing_party.name} party for you, in the #{@swap_with.constituency.name} constituency.
+  In return, you will vote for the #{@swap_with.preferred_party.name} party in your constituency (#{@user.constituency.name}).
 %p
-	We'll send some reminders to #{@swap_with.name} to remind them to vote for your chosen party on December 12th, and we'll ask them for confirmation once they have.
+  We'll send some reminders to #{@swap_with.name} to remind them to vote for your chosen party on December 12th, and we'll ask them for confirmation once they have.
 
-	- if @swap_with.profile_url.present?
-		In the meantime, why not
-		%a{href: @swap_with.profile_url} reach out to #{@swap_with.name} directly?
+  - if @swap_with.profile_url.present?
+    In the meantime, why not
+    %a{href: @swap_with.profile_url} reach out to #{@swap_with.name} directly?
```

Comment:
No, you can't assume we have their social profile - see #192.  I see that this is actually a regression I missed :-(  I think it was introduced as part of the monster email login PR.